Gio-2.0.gir

```diff
@@ -74098,6 +74098,19 @@ g_tls_certificate_new_from_pem().</doc>
           </parameter>
         </parameters>
       </constructor>
+      <constructor name="new_from_file_with_password" c:identifier="g_tls_certificate_new_from_file_with_password" throws="1">
```

Comment:
Docs have `Since: 2.72` but that doesn't end up here. @bilelmoussaoui we had that before, do you remember where that was reported?

Reply:
I opened https://gitlab.gnome.org/GNOME/gobject-introspection/-/issues/412 for that